Files/BaseLayout.cs

```diff
@@ -484,8 +483,9 @@ protected override void OnNavigatingFrom(NavigatingCancelEventArgs e)
             var parameter = e.Parameter as NavigationArguments;
             if (!parameter.IsLayoutSwitch)
             {
-                ParentShellPageInstance.FilesystemViewModel.CancelLoadAndClearFiles();
+                await ParentShellPageInstance.FilesystemViewModel.CancelLoadAndClearFiles();
             }
+            GC.Collect();
```

Comment:
This was placed here for testing only

Reply:
@gave92 Do you think we should remove this from production code?

Reply:
@gave92 Do you think we should remove this from production code?

Reply:
Yeah I think so